code/lib/cli/src/scaffolding-readme-template.md

```diff
@@ -0,0 +1,23 @@
+# {{template.name}}
```

Comment:
Hey @kylegach you might be interested in this. This would be the README file for people who initialize  Storybook in empty projects